### ISSUE TITLE
i can not reprodu

### ISSUE THREAD

**Issue by Dlemand** [EadCat/APAP-Image-Stitching#5]

**Comment by Dlemand** [comment]
i run this code and use the images of the railway offered by the paper .well, the orbits fits good but the Tower crane in the backround exists some error.e imag
![out_00000](https://user-images.githubusercontent.com/51884893/147480126-bab23914-d077-42f1-98a9-b8849b298a82.jpg)

**Comment by EadCat** [comment]
I fixed one line in the code to obtain local homography as follows:

<img width="893" alt="20220105012835" src="https://user-images.githubusercontent.com/59416168/148094448-5701e2b1-1f36-4c34-bd18-4d6d740f65a0.png">

After that, I tested the code by modifying the following hyperparameters several times.
1. RANSAC threshold
2. Random sampling number for RANSAC
3. gamma (in the paper)
4. sigma (in the paper)

For example, in a situation where the input image is resized to a size 1000×750, I got this image.

![05-thr35-m16-ge5-s8](https://user-images.githubusercontent.com/59416168/148095142-9ec603a4-b1e2-48f1-a355-e0b817a61507.jpg)
1 -> 35 / 2 -> 16 / 3-> 0.00005 / 4-> 8, Forced to execute RANSAC 500 times.

As written in the paper, these hyperparameters should be set differently for each size. For example, if you experiment with the same hyperparameters to create an image in a size 2000x1500, you will get the following image.

![06-same05-2000x1500](https://user-images.githubusercontent.com/59416168/148095331-748a5f2b-f4f5-47ca-990f-732ca5f3cdc3.jpg)

Unfortunately, I've done a few experiments, but I couldn't get a perfect image like the paper with this code.

With the conclusion of the above experiment, hyperparameter setting according to image and size is very important, and I feel that this code needs to be modified so that hyperparameters can be input by option.

I will modify the option ASAP.

Thank you.
